CMakeLists.txt

```diff
@@ -289,7 +289,7 @@ if(GENERATE_DOC)
     file(COPY ${SOURCE_DIR}/docs/aliases DESTINATION ${BUILD_DIR}/docs)
 
     add_custom_target(ldoc ALL
-        DEPENDS ${BUILD_DIR}/doc/index.html)
+        DEPENDS generate-examples ${BUILD_DIR}/doc/index.html)
```

Comment:
Why exactly is this needed? And why is this dependency here and not on the `add_custom_command` in line 300?

Reply:
It is not really needed currently, but it is a dependency.
I plan to e.g. not enable/run ldoc by default, so this describes the depency in this case (when not everything is added to ALL).
Will move it to line 300 though.